lua/kubectl/state.lua

```diff
@@ -89,8 +89,8 @@ function M.checkVersions()
   commands.shell_command_async("kubectl", { "version", "--output", "json" }, function(data)
     local result = decode(data)
     if result then
-      local clientVersion = result.clientVersion.gitVersion
-      local serverVersion = result.serverVersion.gitVersion
+      local clientVersion = result and result.clientVersion and result.clientVersion.gitVersion or "0.0"
```

Comment:
Is the first check (result) needed? Don't we check that in the line above?

Reply:
you are right